workflow/daymet.py

```diff
@@ -263,6 +278,12 @@ def daymetToATS(dat, smooth=False, smooth_filter=False, nyears=None):
     dout['relative humidity [-]'] = np.minimum(1.0, dat['vp']/sat_vp_Pa) # -
     dout['precipitation rain [m s^-1]'] = np.where(mean_air_temp_c >= 0, precip_ms, 0)
     dout['precipitation snow [m SWE s^-1]'] = np.where(mean_air_temp_c < 0, precip_ms, 0)
+    dout['wind speed [m s^-1]'] = 4. * np.ones_like(dout['relative humidity [-]'])
```

Comment:
I removed wind speed from our DayMet files on purpose previously.  Wind speed is not in DayMet -- adding it to the DayMet file is confusing, and reading wind speed this way rather than just setting it to a constant value in the input file is more expensive.  Please remove this.